profiles/sway.py

```diff
@@ -1,7 +1,7 @@
 # A desktop environment using "Sway"
+from archinstall import Menu
```

Comment:
Technically you don't even need this import if you do `archinstall.Menu()` in the code :) saves one import

Reply:
True that :) 
I've removed the imports now and access it directly with `archinstall.Menu`